lib/fxpay/settings.js

```diff
@@ -10,6 +10,9 @@
     //
     // Disallow receipts belonging to other apps.
     allowAnyAppReceipt: false,
```

Comment:
Oh my comment was wrong, I thought it was flipped the other way. Leave as is. Sorry for the noise.